drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -463,6 +504,7 @@ function DragAndDropTemplates(configuration) {
         return (
             h('section.themed-xblock.xblock--drag-and-drop', [
                 problemTitle,
+                h('div.problem-progress', {key: progress_id, id: progress_id}, progress_text),
```

Comment:
@itsjeyd The `key` entry does not translate to HTML, it's an internal property used by virtual-dom so that it can distinguish elements when reordering them.
For example if you first render two elements `[h('div', 'a'), h('div', 'b')]`, and then reorder them when rendering the second time (`[h('div', 'b'), h('div', 'a')]`), virtual-dom will not be able to know that the elements have been reordered so it will patch the two corresponding DOM elements rather than reordering them. It usually doesn't matter since the visual end result is the same, but if the two DOM elements contain any internal state (for example if you modify then with jQuery outside of virtual-dom), it can lead to weird issues.

There is no reordering going on here, so the `key` property isn't really needed. I added it while trying to fix weird selenium failures just in case it helped (it didn't; the issues were not caused by virtual-dom). I will remove the `key` property.